lib/fog/vsphere/requests/compute/vm_clone.rb

```diff
@@ -768,6 +769,20 @@ def modify_template_volumes_specs(vm_mob_ref, volumes)
           specs.concat(new_volumes.map { |volume| create_disk(volume) })
           return specs
         end
+
+        def relocate_template_volumes_specs(vm_mob_ref, volumes, datacenter)
+          template_volumes = vm_mob_ref.config.hardware.device.grep(RbVmomi::VIM::VirtualDisk)
+          modified_volumes = volumes.take(template_volumes.size)
+          new_volumes      = volumes.drop(template_volumes.size)
```

Comment:
It doesn't appear that `new_volumes` is used anywhere. Is this correct? Is there a typo in the block below or this variable unnecessary?

Reply:
You're right, I added new volumes to relocate spec, but it failed and also it isn't necessary, because they are created in right datastore, I will remove that line